### PR TITLE
[test] Make parameters loggable in Amber and Gromacs tests

### PR DESCRIPTION
**Pull request by vkarak**
I have also aligned the implementation of the Amber library test with that of Gromacs.

### CODE REVIEW

File: hpctestlib/sciapps/amber/nve.py [hunk 1]
```diff
@@ -107,9 +80,35 @@ class amber_nve_check(rfm.RunOnlyRegressionTest, pin_prefix=True):
...
+
+        :type: :class:`str`
+        '''
+        return self.__nrg_tol
+
     @run_after('init')
     def prepare_test(self):
-        self.benchmark, self.energy_ref, self.energy_tol = self.benchmark_info
+        self.__bench, self.__nrg_ref, self.__nrg_tol = self.benchmark_info
         self.descr = f'Amber NVE {self.benchmark} benchmark ({self.variant})'
```
Comment (teojgo) [inline hpctestlib/sciapps/amber/nve.py:112]:
```suggestion
        self.descr = f'Amber NVE {self.bench_name} benchmark ({self.variant})'
```

**Review by teojgo — CHANGES_REQUESTED** [review]

File: hpctestlib/sciapps/amber/nve.py (hunk 1 above)
Comment (teojgo) [reply hpctestlib/sciapps/amber/nve.py]: The same change is needed also on line 126.

**Review by victorusu — APPROVED** [review]
`lgtm`

**Comment by vkarak** [comment]
@jenkins-cscs retry dom

**Review by teojgo — APPROVED** [review]

**Pull request merged**